src/main.cpp

```diff
@@ -3996,14 +3996,18 @@ std::string GetWarnings(const std::string& strFor)
     if (fLargeWorkForkFound)
     {
         nPriority = 2000;
-        strStatusBar = strRPC = _("Warning: The network does not appear to fully agree! Some miners appear to be experiencing issues.");
+        strStatusBar = strRPC = "Warning: The network does not appear to fully agree! Some miners appear to be experiencing issues.";
     }
     else if (fLargeWorkInvalidChainFound)
     {
         nPriority = 2000;
-        strStatusBar = strRPC = _("Warning: We do not appear to fully agree with our peers! You may need to upgrade, or other nodes may need to upgrade.");
+        strStatusBar = strRPC = "Warning: We do not appear to fully agree with our peers! You may need to upgrade, or other nodes may need to upgrade.";
     }
 
+    // try to i18n the string
+    if (translate)
+        strStatusBar = _(strStatusBar.c_str());
```

Comment:
> The translations will go away.

Hmm... what exactly do you mean with this?

If I'm right, `strStatusBar = _(strStatusBar.c_str());` should translate the string if possible, and if not, keep the existing string. Future changes of the original string or of translations should not be a problem.

Or am I blind somewhere?